app/settings.py

```diff
@@ -1,14 +1,52 @@
 import os
 import typing as tp
 
-from pathlib import Path
-
-from dotenv import load_dotenv
-
-env_path = Path('.env')
-load_dotenv(dotenv_path=env_path)
-
 # FLASK SETTINGS
 SECRET_KEY: tp.Optional[str] = os.environ.get('SECRET_KEY')
 FLASK_APP: tp.Optional[str] = os.environ.get('FLASK_APP')
 FLASK_ENV: tp.Optional[str] = os.environ.get('FLASK_ENV')
+
+# DB SETTINGS
+DATABASE: tp.Optional[str] = os.environ.get('DATABASE')
+POSTGRES_HOST: tp.Optional[str] = os.environ.get('RDS_HOSTNAME')
+POSTGRES_USER: tp.Optional[str] = os.environ.get('RDS_USERNAME')
+POSTGRES_PASSWORD: tp.Optional[str] = os.environ.get('RDS_PASSWORD')
+POSTGRES_DB: tp.Optional[str] = os.environ.get('RDS_DB_NAME')
+POSTGRES_PORT: tp.Optional[str] = os.environ.get('RDS_PORT')
+
+# TEST DB SETTINGS
+TEST_POSTGRES_HOST: tp.Optional[str] = 'db-test'
+TEST_POSTGRES_USER: tp.Optional[str] = 'test_user'
+TEST_POSTGRES_PASSWORD: tp.Optional[str] = 'test_password'
+TEST_POSTGRES_DB: tp.Optional[str] = 'test_db'
+TEST_POSTGRES_PORT: tp.Optional[str] = '5432'
+
+# JWT SETTINGS
+ACCESS_TOKEN_EXPIRE_MINUTES: tp.Optional[int] = int(os.environ.get('ACCESS_TOKEN_EXPIRE_MINUTES'))  # type: ignore
+REFRESH_TOKEN_EXPIRE_MINUTES: tp.Optional[int] = int(os.environ.get('REFRESH_TOKEN_EXPIRE_MINUTES'))  # type: ignore
+DEFAULT_ACCESS_TOKEN_EXPIRE_MINUTES = 180
+DEFAULT_REFRESH_TOKEN_EXPIRE_MINUTES = 360
+TOKEN_TYPE: tp.Optional[str] = os.environ.get('TOKEN_TYPE')
+TOKEN_ALGORITHM: tp.Optional[str] = os.environ.get('TOKEN_ALGORITHM')
+ACCESS_TOKEN_JWT_SUBJECT: tp.Optional[str] = os.environ.get('ACCESS_TOKEN_JWT_SUBJECT')
+REFRESH_TOKEN_JWT_SUBJECT: tp.Optional[str] = os.environ.get('REFRESH_TOKEN_JWT_SUBJECT')
+
+# AWS SETTINGS
+AWS_ACCESS_KEY_ID: tp.Optional[str] = os.environ.get('AWS_ACCESS_KEY_ID')
+AWS_SECRET_ACCESS_KEY: tp.Optional[str] = os.environ.get('AWS_SECRET_ACCESS_KEY')
```

Comment:
best practice is not to have access keys in application. rather grant appropriate permissions to your servers, like access to s3

Reply:
https://aws.amazon.com/premiumsupport/knowledge-center/elastic-beanstalk-s3-bucket-instance/

Reply:
done